src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int hypotenuse;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg, int hypotenuse) {

```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for `firstLeg`, `secondLeg`, and `hypotenuse` and pass them on for better code readability. Also, it's a good practice to validate the arguments to ensure they create a valid right triangle.